index.xml

```diff
@@ -8390,7 +8382,7 @@ context.setColorClamped(-1, 255, 257);</x:codeblock>
             <p>
               The prose definition of a constructor <span class='rfc2119'>MUST</span>
               either return an IDL value of a type corresponding to the interface
-              or dictionary the <a class='xattr' href='#Constructor'>[Constructor]</a>
+              the <a class='xattr' href='#Constructor'>[Constructor]</a>
               extended attribute appears on, or throw an exception.
             </p>
             <p>
```

Comment:
Can you get rid of the conditional in the paragraph that begins on this line (sorry, bit hard to link to) so that it just disallows [NoInterfaceObject] when [Constructor] is used any time.